sentry/workflows.py

```diff
@@ -0,0 +1,16 @@
+from temporalio import activity, workflow
+from dataclasses import dataclass
+from datetime import timedelta
+from sentry.activities import ComposeGreetingInput, compose_greeting
```

Comment:
```suggestion
# Import our activity and param class, passing it through the sandbox
with workflow.unsafe.imports_passed_through():
    from sentry.activities import ComposeGreetingInput, compose_greeting
```

You want to pass this through the sandbox, you don't want to reload it everytime